### DISCUSSION TITLE
The php file is deleted

### DISCUSSION THREAD

**Discussion by YAcostaDev** [prasathmani/tinyfilemanager#863] [Q&A]
It's an excellent project, too bad I can't use it because the php programming code is completely erased, find out and it's because my hosting uses an antivirus and detects it as malware software. Is there a way to overcome that?

ORIGINAL:
Es un excelente proyecto, lastima que no lo puedo usar porque el código de programación del php se borra completamente, averigüe y es porque mi hosting usa un antivirus y lo detecta como software malware. ¿Habría forma de poder superar eso?

**Comment by prasathmani** [comment]
#578 , #273

**Reply by cracada** [reply]
I also get a message from host: BAILOUT malware detected

**Comment by SamSirry** [comment]
> ¿Habría forma de poder superar eso?

If your hosting provider support it, they will allow you to exclude this file from the scanner, otherwise change your hosting provider.

**Comment by jharbin** [comment]
I have the same issue.  I use Hostinger.com as my host and when I couldn't access my file location, in looking in the HPanel interface, the Malware Scanner flagged and deleted both of the index.php (renamed TFM) files, as malware.  Is that particular function or feature that we could trim/remove/disable to keep these from flagging?

**Reply by jharbin** [reply]
It doesn't seem like Hostinger allows for a whitelist of these files.  Not sure how many other hosting companies are in the same predicament.
